### ISSUE TITLE
i18n 'loadmessags' command fails with error

### ISSUE THREAD

**Issue by chrabyrd** [archesproject/arches#8706]
```
python manage.py i18n loadmessages --lang=en
System check identified some issues:

WARNINGS:
Traceback (most recent call last):
  File "manage.py", line 29, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/cbyrd/Projects/ENV/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/Users/cbyrd/Projects/ENV/lib/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/cbyrd/Projects/ENV/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/cbyrd/Projects/ENV/lib/python3.8/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/Users/cbyrd/Projects/arches/arches/management/commands/i18n.py", line 48, in handle
    self.load_messages(lang=options["lang"])
  File "/Users/cbyrd/Projects/arches/arches/management/commands/i18n.py", line 60, in load_messages
    ArchesPOLoader(file.file, settings.LANGUAGE_CODE, file.language).load()
AttributeError: 'tuple' object has no attribute 'file'
```
```
> files
[('en', [], False)]
```

**Comment by chrabyrd** [comment]
closing as user error for now

**Comment by chrabyrd** [comment]
resolved in #8711
